src/hypervisor.c

```diff
@@ -146,6 +146,11 @@ cc_oci_expand_cmdline (struct cc_oci_config *config,
 		goto out;
 	}
 
+	if (! config->bundle_path) {
```

Comment:
nitpick: Did you sneak this bundle_path check in at the same time as the tty fix? Just it confused me a little as the only change to this file.